src/UI/templates/default/Icon/icon.less

```diff
@@ -0,0 +1,67 @@
+.icon {
+	background-repeat: no-repeat;
+	background-size: cover;
+	background-image: url('./images/icon_default.svg');
```

Comment:
I don't see why this should be a less variable. The standard way to exchange icons in a skin would be to put a file with the same name in the images-folder of the skin. Why would we want to have two different strategies for the same effect?